src/python_be.cc

```diff
@@ -1389,6 +1390,7 @@ ModelInstanceState::ProcessRequests(
         gpu_buffers_handle_shm[index] = pb_memory->ShmHandle();
         ++index;
       }
+      response_index++;
```

Comment:
What is the use of `response_index`?

Reply:
It is used in Line 1382 for error reporting.